zebrad/src/commands/start/sync.rs

```diff
@@ -425,6 +445,26 @@ where
 
         Ok(())
     }
+
+    /// Get the depth of hash in the current chain.
+    ///
+    /// Returns None if the hash is not present in the chain.
+    /// TODO: handle multiple tips in the state.
+    #[instrument(skip(self))]
+    async fn get_depth(&mut self, hash: BlockHeaderHash) -> Result<Option<u32>, Report> {
```

Comment:
might want a newtype for this

```suggestion
    async fn get_depth(&mut self, hash: BlockHeaderHash) -> Result<Option<block::Depth>, Report> {
```

Reply:
I'd rather just drop the unused depth.

Reply:
See fd61aa9cbbcddd25d98b053215b6460aaee11c0e for a change that drops the unused depth.